jk-barcode-demo/package.json

```diff
@@ -18,6 +18,7 @@
     "@angular/platform-browser": "^15.2.0",
     "@angular/platform-browser-dynamic": "^15.2.0",
     "@angular/router": "^15.2.0",
+    "bwip-js": "^3.4.3",
```

Comment:
You only need this because the library is installed using file://, you shouldnt when installed from npm or a tarball